less/home-page.less

```diff
@@ -25,51 +37,94 @@
       border-radius: 20px;
       height: 100%;
     }
+
+    @media @phone {
+
+      .blast{
+        width: 100%;
+        height: 100%;
+      }
+
+      .sky{
+        display: none;
+      }
+
+      .launch{
+        display: none;
+      }
+
+    }
+
   }
 
-  .visit-planets {
-    padding: 15px;
+}
+
+.visit-planets {
+  padding: 15px;
+
+  h3 {
+    margin-top: 11px;
+  }
+
+  p {
+    margin-top: 13px;
+  }
+}
+
+.walk {
+  margin-top: 60px;
+  border-top: 1px solid @main-border-color;
+  padding: 50px 60px;
+  display: flex;
+  justify-content: center;
+
+  .walk-text {
+    width: 50%;
 
     h3 {
-      margin-top: 11px;
+      margin-bottom: 10px;
     }
 
     p {
-      margin-top: 13px;
+      margin-bottom: 26px;
     }
-  }
-
-  .walk {
-    margin-top: 60px;
-    border-top: 1px solid @main-border-color;
-    padding: 50px 60px;
-    display: flex;
-    justify-content: center;
 
-    .walk-text {
-      width: 50%;
+    @media @phone {
```

Comment:
I think there's an example somewhere, where the media query is below the nested styles, but I think it's actually better practice to nest it directly under the regular styles so line 80.